orm/src/main/java/org/hibernate/search/query/hibernate/impl/FullTextQueryImpl.java

```diff
@@ -176,7 +181,7 @@ private Loader getProjectionLoader(ObjectLoaderBuilder loaderBuilder) {
 	}
 
 	@Override
-	public ScrollableResultsImplementor scroll() {
+	public ScrollableResultsImpl scroll() {
```

Comment:
this was required when I originally made this for ORM 5.2.0, seems I can switch back to `ScrollableResultsImplementor` today with 5.2.1.. thanks.